### ISSUE TITLE
Inpainting sample.py script inpaints entire image

### ISSUE THREAD

**Issue by jeffz0** [Jack000/glid-3-xl-stable#25]
Hi Jack,

Great work with the training scripts. I'm trying to run your pretrained inpainting model with sample.py, but the output just inpaints the entire image rather than the specified region. In your wiki, you show some great examples of inpainting on the face. Do you have a script to produce those results?

Thanks again,

Jeff

**Comment by jeffz0** [comment]
Sorry seems I need to flip the mask I was inputting. However, my problem now is no matter what prompt I give, the generated image ignores the text and tries to just simply fill in the image. I noticed in your wiki you had a denoising strength and I'm guessing this adjustment varies the strength of the text and noise. Where do I adjust this variable? 

Thanks again,

Jeff
